### DISCUSSION TITLE
Can plotting directions  be changed in MTEX 6.0 beta3?

### DISCUSSION THREAD

**Discussion by zstar-kkk** [mtex-toolbox/mtex#2187] [Ask Anything]
Hi MTEX community,

I find that plotting directions can not be changed in MTEX 6.0 beta3? Is it a bug?

Thanks in advance for your help.

**Comment by ralfHielscher** [comment]
Could you please give some more details?

**Reply by zstar-kkk** [reply]
This is my code.
```
% crystal symmetry
CS = {... 
  'notIndexed',...
  crystalSymmetry('432', [2.9 2.9 2.9], 'mineral', 'Iron (Alpha)', 'color', [0.53 0.81 0.98])};

% plotting convention
setMTEXpref('xAxisDirection','east');
setMTEXpref('zAxisDirection','intoPlane');

%% Specify File Names

% path to files
pname = 'H:\exampledata';

% which files to be imported
fname = [pname '\InSituSS_00.osc'];

%% Import the Data

% create an EBSD variable containing the data
ebsd = EBSD.load(fname,CS,'interface','osc');

%% Correct Data

rot = rotation.byEuler(270*degree,180*degree,0*degree);
ebsd = rotate(ebsd,rot,'keepXY');

%%
ipfkey = ipfTSLKey(ebsd.CS);
figure;
plot(ebsd,ipfkey.orientation2color(ebsd.orientations),'coordinates','on')

odf = calcDensity(ebsd.orientations)
figure;
plotPDF(odf,Miller(1,0,0,ebsd.CS))
```
![1](https://github.com/user-attachments/assets/c8bffd85-72d9-4cbf-a863-f6157a628e98)
![2](https://github.com/user-attachments/assets/135ed1c9-e5f0-44df-aff0-1add918b2f1d)


The axis directions are not consistant with the results of commecial software
![3](https://github.com/user-attachments/assets/9983b916-897e-4957-8d97-2695bdb17f14)
![4](https://github.com/user-attachments/assets/6cb9f180-68e6-449e-afca-41ec1c12cdd8)

**Reply by zstar-kkk** [reply]
Should i delet the option "keepxy"?
